server/service/osquery_utils/queries.go

```diff
@@ -186,17 +187,18 @@ var hostDetailQueries = map[string]DetailQuery{
 				return nil
 			}
 
-			version := rows[0]["version"]
+			version := rows[0]["display_version"]
 			if version == "" {
 				level.Debug(logger).Log(
-					"msg", "unable to identify windows version",
+					"msg", "unable to identify windows display version",
```

Comment:
Isn't this expected to be empty for some "products"?

E.g. 11926 and 11927 below: 
```
{
  "ProductName": "Windows 11",
  "Products": {
    "11926": "Windows 11 for x64-based Systems",
    "11927": "Windows 11 for ARM64-based Systems",
    "12085": "Windows 11 Version 22H2 for ARM64-based Systems",
    "12086": "Windows 11 Version 22H2 for x64-based Systems",
    "12242": "Windows 11 Version 23H2 for ARM64-based Systems",
    "12243": "Windows 11 Version 23H2 for x64-based Systems"
  },
```

Reply:
That's a great question.  I think some testing is needed here if we can get our hands on that older build of Win11.    I just looked it up [here[(https://en.wikipedia.org/wiki/Windows_11_version_history) which is saying that build SHOULD be 21H2 🤔

Reply:
I removed the debug message.  As discussed, there may be empty "display_version"s, in which case we'll match against the 1st 2 entries in the products list.